### PR TITLE
Fix contextual docs & improve UX for debug mode

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**
https://github.com/gravitee-io/issues/issues/7323

**Description**

Add copy do clipboard for debug inspector data
Fix contextual docs

**Additional context**

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-orlwmehvqa.chromatic.com)

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/debug-improvement/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/management.module.ajs.ts [hunk 1]
```diff
@@ -702,9 +702,6 @@ graviteeManagementModule.config(($mdThemingProvider: angular.material.IThemingPr
 });
 graviteeManagementModule.config(($showdownProvider) => {
   $showdownProvider.setOption('tables', true);
-  $showdownProvider.loadExtension('highlightjs');
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/management/management.module.ajs.ts:705]: believe me I couldn't find a simpler way 😁 

The extensions seem to load autonomously even if they are not explicitly loaded here 🤷‍♂️ 
⚠️ If you know how to check each extension. Don't hesitate to tell me, I didn't really find it. I did several tests and I think it's good but I'm not sure for `prettify` one

**Review by gcusnieux — APPROVED** [review]

File: gravitee-apim-console-webui/src/shared/components/gio-diff/gio-diff.component.spec.ts [hunk 2]
```diff
@@ -86,7 +86,7 @@ describe('GioDiffComponent', () => {
   });
 
   describe('without diff between left and right', () => {
-    it('should force raw format and display only one gv-code', async () => {
+    fit('should force raw format and display only one gv-code', async () => {
```
Comment (gaetanmaisse) [inline gravitee-apim-console-webui/src/shared/components/gio-diff/gio-diff.component.spec.ts]: :aa:

**Review by gaetanmaisse — CHANGES_REQUESTED** [review]

File: gravitee-apim-console-webui/src/shared/components/gio-diff/gio-diff.component.spec.ts (hunk 2 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/shared/components/gio-diff/gio-diff.component.spec.ts]: :aa:

File: gravitee-apim-console-webui/src/management/management.module.ajs.ts (hunk 1 above)
Comment (gaetanmaisse) [reply gravitee-apim-console-webui/src/management/management.module.ajs.ts:705]: Ok, let's go with that then, I 💯  trust you 👍🏻

**Review by gaetanmaisse — APPROVED** [review]

**Pull request merged**